web/app/scripts/stepwise/stepwise-directive.js

```diff
@@ -4,154 +4,302 @@
  *  the property 'occurred_from'.
  */
 
-(function () {
+(function() {
     'use strict';
 
     /* ngInject */
-    function Stepwise($translate, LanguageState) {
+    function Stepwise($translate, $window, LanguageState) {
+        var inMapMode = false;
         var module = {
             restrict: 'E',
             scope: {
-              chartData: '=',
-              minDate: '=',
-              maxDate: '='
+                chartData: '=',
```

Comment:
Recent change to our preferred style, but it would be good to switch these to one-way bindings if possible.